components/variant-summary.js

```diff
@@ -91,88 +92,134 @@ export default function VariantSummary({
+                    normalizedTissueSpecificScore={
+                      normalizedTissueSpecificScore
+                    }
+                    colorBy={"Colored by tissue specific score"}
+                    width={"w-10/12"}
                   />
-                )}
-              </div>
-              <div className="absolute top-12 right-6 h-48">
-                <TissueScoreBar
-                  normalizedTissueSpecificScore={normalizedTissueSpecificScore}
-                />
+                  {organFilters.length > 0 && (
+                    <Selections
+                      filters={organFilters}
+                      clearFilterFunc={handleClickOrgan}
+                    />
+                  )}
+                </div>
+                <div className="absolute top-12 right-6 h-48">
+                  <TissueScoreBar
+                    normalizedTissueSpecificScore={
+                      normalizedTissueSpecificScore
+                    }
+                  />
+                </div>
               </div>
-            </div>
-          </DataPanel>
-          <DataPanel className="col-span-2">
+            </DataPanel>
+            <DataPanel className="col-span-2">
+              <DataArea>
+                <DataItemLabel>Searched Coordinates</DataItemLabel>
+                <DataItemValue>{data.query_coordinates[0]}</DataItemValue>
+                <DataItemLabel>Genome Assembly</DataItemLabel>
+                <DataItemValue>{data.assembly}</DataItemValue>
+                <DataItemLabel>Global Rank</DataItemLabel>
+                <DataItemValue>{data.regulome_score.ranking}</DataItemValue>
+                <DataItemLabel>Global Score</DataItemLabel>
+                <DataItemValue>{data.regulome_score.probability}</DataItemValue>
+                <DataItemLabel>Tissue Specific Scores</DataItemLabel>
+                <div className="w-11/12">
+                  <Sparkline
+                    scores={data.regulome_score.tissue_specific_scores}
+                    maxBarThickness={8}
+                    thumbnail
+                  />
+                </div>
+                {data.variants[0].spdi && (
+                  <>
+                    <DataItemLabel>SPDI</DataItemLabel>
+                    <DataItemValue>{data.variants[0].spdi}</DataItemValue>
+                    <DataItemLabel>HGVS</DataItemLabel>
+                    <DataItemValue>{data.variants[0].hgvs}</DataItemValue>
+                    <DataItemLabel>Ref</DataItemLabel>
+                    <DataItemValue>{data.variants[0].ref}</DataItemValue>
+                    <DataItemLabel>Alt</DataItemLabel>
+                    <DataItemValue>{data.variants[0].alt}</DataItemValue>
+                  </>
+                )}
+                {data.variants[0].rsids.length > 0 && (
+                  <>
+                    <DataItemLabel>rsID</DataItemLabel>
+                    <DataItemValue>{data.variants[0].rsids}</DataItemValue>
```

Comment:
Is `data.variants[0].rsids` an array? Checking its length > 0 makes it sound like one. Does `{data.variants[0].rsids}` need a join?

Reply:
You are correct. Will update it.